.travis.yml

```diff
@@ -25,7 +24,7 @@ before_script:
   - Xvfb :99 -screen 0 1024x768x24 -extension RANDR &
 
 script:
-    - xvfb-maybe npm run build
+    - xvfb-maybe npm install
```

Comment:
Not sure how I feel about the tests being in postbuild, I don't feel like tests are part of building I think those should be separated.